packages/styles/package.json

```diff
@@ -38,7 +38,7 @@
     "@carbon/import-once": "10.3.0",
     "@carbon/layout": "10.13.0",
     "@carbon/motion": "10.9.0",
-    "@carbon/themes": "10.20.0",
+    "@carbon/themes": "10.21.0",
     "@carbon/type": "10.16.0",
     "carbon-components": "10.21.0"
```

Comment:
@kennylam it doesn't look like all components were updated, did you run the script `yarn upgrade-carbon` to get all of the carbon dependencies?

Reply:
@jeffchew I did. not sure why that one didn't get picked up